config/webpack.config.js

```diff
@@ -23,6 +23,7 @@ const ForkTsCheckerWebpackPlugin = require('react-dev-utils/ForkTsCheckerWebpack
 const typescriptFormatter = require('react-dev-utils/typescriptFormatter');
 const postcssNormalize = require('postcss-normalize');
 const miniExtract = require('mini-css-extract-plugin');
+const camelCase = require('camelcase');
```

Comment:
good enough for the moment, but if you I can take a look and have our own utility based on ramda, such that we'd have one less dependency:

https://gist.github.com/yurifrl/7061e440afea1c01e8d3f421832cb508

Reply:
That package is already used somewhere else and it's only in development tools.